ulauncher/utils/fuzzy_search.py

```diff
@@ -48,19 +48,24 @@ def get_matching_indexes(query, text):
     return sorted(positions)
 
 
-def get_score(query, text):
+def get_score(query="", text=""):
```

Comment:
defaulting both args seems to indicate it is meaningful and expected to call as `get_score()`, or maybe call as `get_score(text="something")`. While it will return a score of `0` for `get_score()`, which is reasonable in some sense, I'm not sure it really matches with the semantics of the function. Would it not be better to error if this was ever called with anything other than exactly 2 arguments?

Reply:
Good point. I can't even remember why I did that.

Reply:
It didn't break any behavior or test, so pushed it as 7f720f6